tutorials/abap-s4hcloud-procurement-po-customlogic-tracing/abap-s4hcloud-procurement-po-customlogic-tracing.md

```diff
@@ -1,86 +1,89 @@
 ---
-title: Create Custom Logic Using Key User Extensibility and Perform Trace
-description: Create custom logic for an existing custom field in S/4HANA Cloud ABAP Environment, then check its parameters using the Custom Logic Tracing app
+parser: v2
 auto_validation: true
 time: 15
-tags: [ tutorial>intermediate, programming-tool>abap-development, software-product>sap-business-technology-platform, software-product>sap-btp--abap-environment, software-product>sap-s-4hana-cloud]
+tags: [ tutorial>intermediate, programming-tool>abap-development, software-product>sap-business-technology-platform, software-product>sap-btp--abap-environment]
 primary_tag: software-product-function>s-4hana-cloud-abap-environment
 author_name: Julie Plummer
 author_profile: https://github.com/julieplummer20
 
 ---
 
 
+# Create Custom Logic Using Key User Extensibility and Perform Trace
+<!-- description --> Create custom logic for an existing custom field in S/4HANA Cloud ABAP Environment, then check its parameters using the Custom Logic Tracing app
+
 ## Prerequisites
 - You have a license for SAP S/4HANA Cloud
 - You are familiar with Key User extensibility in SAP S/4HANA Cloud. If not, see the **More Information** section
 - You are familiar with the app **Custom Logic Tracing**. If not, see the **More Information** section
 - In SAP Fiori launchpad, you have the following authorizations:
     - **`SAP_BR_PURCHASER`** for the app **Create Purchase Order - Advanced**
     - **`SAP_BR_ADMINISTRATOR`** for **Custom Logic Tracing** and **Custom Logic**
-- You have created a custom field using Key User extensibility **`yy1_zhdrprnt2_pdh`**
-- In Custom Logic, you have created a `BAdI` implementation `YY1_FILLHEADERCUSTOMFIELDS` in the business context and `BAdI` definition **Modify Purchase Order Header (`MM_PUR_S4_PO_MODIFY_HEADER`)**
+- You have created a custom field using Key User extensibility **`yy1_zhdrprnt2_pdh_000`**
+- In Custom Logic, you have created a `BAdI` implementation `YY1_FILLHEADERCUSTOMFIELDS_000` in the business context and `BAdI` definition **Modify Purchase Order Header (`MM_PUR_S4_PO_MODIFY_HEADER`)**
 
 
 
-## Details
-### You will learn
+## You will learn
 - How to create custom logic for an existing custom field in the app **Custom Fields**
 - How to run a trace for the Fiori app **Create Purchase Order** in the app **Custom Logic Tracing**
 - How to check the output value of this custom field using **Custom Logic Tracing**
 - How to check the custom logic of this custom field, in **Custom Fields**
 
+## Intro
+>**NOTE**: The administrator receives an welcome e-mail after provisioning. This e-mail includes the system URL. By removing `/ui` you can log into the SAP S/4HANA Cloud ABAP Environment system. Further information can be found [Developer Extensibility: Connect to the ABAP System](https://help.sap.com/docs/SAP_S4HANA_CLOUD/6aa39f1ac05441e5a23f484f31e477e7/4b962c243a3342189f8af460cc444883.html?locale=en-US).
```

Comment:
Please do not start a note with the label "NOTE".
Please check the style guide for details: https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtqfqchq21